6.0/core/defining_properties.md

```diff
@@ -1,17 +1,8 @@
 # Properties in more detail
 
-<div class="note">
 
-<div class="admonition-title">
-
-Note
-
-</div>
-
-Please take a look at `property_features` first to get an compact overview of
-the available features.
-
-</div>
+> :memo: Please take a look at `property_features` first to get an compact overview of
```

Comment:
`first to get an compact overview` -> `first, to get a compact overview`
added comma and replaced "an" with "a"